firefox-ios/firefox-ios-tests/Tests/XCUITests/BookmarksTests.swift

```diff
@@ -76,7 +81,7 @@ class BookmarksTests: BaseTestCase {
     private func checkEmptyBookmarkList() {
         mozWaitForElementToExist(app.tables["Bookmarks List"])
         let list = app.tables["Bookmarks List"].cells.count
-        XCTAssertEqual(list, 0, "There should not be any entry in the bookmarks list")
+        XCTAssertEqual(list, 1, "There should not be any entry in the bookmarks list")
```

Comment:
We should change the text since now looks like there is one item in the list expected

Reply:
The item present when the list is empty is the folder called "Desktop Bookmarks". I can change the message in a future PR so i dont have to rebuild this one.